libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/AccessibilityUtils.java

```diff
@@ -26,4 +33,22 @@ public static int getSnackbarDuration(Context ctx, int defaultDuration) {
         return defaultDuration == Snackbar.LENGTH_INDEFINITE ? Snackbar.LENGTH_INDEFINITE
                 : isAccessibilityEnabled(ctx) ? SNACKBAR_WITH_ACTION_DURATION_IN_MILLIS : defaultDuration;
     }
+
+    public static void disableHintAnnouncement(@NonNull TextView textView) {
+        ViewCompat.setAccessibilityDelegate(textView, new AccessibilityDelegateCompat() {
```

Comment:
It might be a nice improvement to check if the view already has a delagate and log an error(possible crash in Debug perhaps?) if it does as we'll override i. Wdyt?

Reply:
This sounds like a good idea so if anyone comes in the future trying to add a delegate there are no weird side effects.

Reply:
Resolved in 30ff588